components/Form/index.js

```diff
@@ -4,29 +4,31 @@ import { useState } from "react";
 import globalPlants from "@/public/data";
 import { useAtom } from "jotai";
 
-export default function Form() {
+export default function Form({ setTrigger }) {
   const [plants, setPlants] = useAtom(globalPlants);
 
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const newPlant = Object.fromEntries(formData);
-    // const newPlant = {
-    //   name: plantName,
-    //   wateringTime: wateringTime,
-    // };
+
     setPlants([...plants, { ...newPlant, id: crypto.randomUUID() }]);
-    console.log(newPlant);
   }
 
   return (
     <>
       <h2>Add a new plant</h2>
-      <form type="add-edit" onSubmit={handleSubmit}>
+      <form
+        type="add-edit"
```

Comment:
what purpose does this typo serve? 🤔

Reply:
I now realized that it had no use 😬 and changed your requests!